doc/command-line.md

```diff
@@ -109,16 +145,199 @@ Options:
 --labels -- display the signature names (by default, the filenames) on the plot
 --indices -- turn off index display on the plot.
 --vmax -- maximum value (default 1.0) for heatmap.
---vmin -- minimum value (deafult 0.0) for heatmap.
+--vmin -- minimum value (default 0.0) for heatmap.
 --subsample=<N> -- plot a maximum of <N> samples, randomly chosen.
 --subsample-seed=<seed> -- seed for pseudorandom number generator.
 ```
-Example figures:
 
-<img src="https://github.com/dib-lab/sourmash/blob/master/doc/_static/cmp.matrix.png?raw=true" style="width:60%" />
+Example output:
+
+![An E. coli comparison plot](_static/ecoli_cmp.matrix.png)
+
+### `sourmash search`
+
+The `search` subcommand searches a collection of signatures or SBTs for
+matches to the query signature.  It can search for matches with either
+high [Jaccard similarity](https://en.wikipedia.org/wiki/Jaccard_index)
+or containment; the default is to use Jaccard similarity, unless
+`--containment` is specified.  `-o/--output` will create a CSV file
+containing the matches.
+
+`search` will load all of provided signatures into memory, which can
+be slow and somewhat memory intensive for large collections.  You can
+use `sourmash index` to create a Sequence Bloom Tree (SBT) that can
+be quickly searched on disk; this is [the same format in which we provide
+GenBank and other databases](databases.html).
+
+Usage:
+```
+sourmash search query.sig [ list of signatures or SBTs ]
+```
+
+Example output:
+
+```
+49 matches; showing first 20:
+similarity   match
+----------   -----
+ 75.4%       NZ_JMGW01000001.1 Escherichia coli 1-176-05_S4_C2 e117605...
+ 72.2%       NZ_GG774190.1 Escherichia coli MS 196-1 Scfld2538, whole ...
+ 71.4%       NZ_JMGU01000001.1 Escherichia coli 2-011-08_S3_C2 e201108...
+ 70.1%       NZ_JHRU01000001.1 Escherichia coli strain 100854 100854_1...
+ 69.0%       NZ_JH659569.1 Escherichia coli M919 supercont2.1, whole g...
+...    
+```
+
+### `sourmash gather`
+
+The `gather` subcommand finds all non-overlapping matches to the
+query.  This is specifically meant for metagenome and genome bin
+analysis.  (See [Classifying Signatures](classifying-signatures.html)
+for more information on the different approaches that can be used
+here.)
+
+If the input signature was computed with `--track-abundance`, output
+will be abundance weighted (unless `--ignore-abundances` is
+specified).  `-o/--output` will create a CSV file containing the
+matches.
+
+`gather`, like `search`, will load all of provided signatures into
+memory.  You can use `sourmash index` to create a Sequence Bloom Tree
+(SBT) that can be quickly searched on disk; this is
+[the same format in which we provide GenBank and other databases](databases.html).
```

Comment:
Line 205-207 is repeated above. I see the utility of the information being in both places, but is it repeated verbatim intentionally?

Reply:
Yep!